interface/login/login.php

```diff
@@ -393,5 +393,28 @@ function imsubmitted() {
             </div>
         </form>
     </div>
+    <script>
+        //Store i18n file in the sessionStorage, replace json on language is changed.
+        //The translations will be available for js in each page
+        //For develop js with translation recommended to load the i18next library with Header::setupHeader(['js-translation'])
+        function loadI18nToSessionStorage(lang_code) {
+            $('[type="submit"]').attr('disabled', true);
+            $.ajax( {
+                url: <?php echo js_escape($GLOBALS['webroot'])?>+"/interface/language/i18n/"+lang_code+".json",
+                dataType: 'text'
+            })
+            .done(function (data) {
+                sessionStorage.setItem('i18n', data);
```

Comment:
I think the key for the sessionStorage will need to have the language name and the "site" directory (usually default; ie. the multisite stuff) in it to avoid weird things happening if the same browser has multiple logins in it (using different languages and/or site).
Also, would it make more sense to run this function when user tries to login rather than every time they change the language in the selector?